danspeech/pretrained_models/folketinget_model.py

```diff
@@ -0,0 +1,16 @@
+from danspeech.deepspeech.model import DeepSpeech
+from danspeech.utils.data_utils import get_model
+
+MODEL_PACKAGE = 'https://github.com/Rasmusafj/models_development/raw/master/400units.pth'
```

Comment:
Dangerous to have a wrong package link, should be toDo I guess?

Reply:
Yep! Changed to todo